BEIMA.Backend.FT/BeimaClient.cs

```diff
@@ -97,10 +98,21 @@ public async Task<HttpResponseMessage> SendRequest(string route, HttpVerb verb,
             switch (verb)
             {
                 case HttpVerb.GET:
-                    response = await _httpClient.GetAsync(route);
```

Comment:
Instead of doing this and creating a new AuthenticationHeaderValue each time we send a request, I would suggest just setting the _httpClient authorization header on the login method:
```httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", "Your Oauth token");```
This way we remove a lot of complexity in having to pass in our token every time we make a request, and just store it in the client object's header itself. We can just login and not have to worry about the token in any of the FTs.

Reply:
That's how I originally had it, but wasn't sure if I should leave it off so that we could have ft tests to check sending without a token. Since we have the mock tests though its probably fine.

Reply:
We could create a new instance of the BeimaClient to test that instead of using TestClient on those tests.

Reply:
Or implement logout functionality?

Reply:
I like the idea of separate clients. I'll go with that one